hypervisor/src/mshv/x86_64/mod.rs

```diff
@@ -38,10 +38,9 @@ pub struct VcpuMshvState {
     pub sregs: MshvSpecialRegisters,
     pub fpu: FpuState,
     pub xcrs: ExtendedControlRegisters,
-    pub lapic: LapicState,
```

Comment:
We should also clean up the code on line 290 which helps in converting LapicState and MshvLapicState.

Reply:
LocalApic is required to stay as is. That's because LAPIC is needed separately during VM configuration.